packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.tsx

```diff
@@ -40,6 +41,8 @@ type TContractDrawerProps = RouteComponentProps & {
         | 'onClickSell'
     >;
 
+const PAGE_BOTTOM_MARGIN = ['iOS', 'unknown'].includes(mobileOSDetect()) ? 0 : 13;
```

Comment:
`clientHeight` is calculated differently on IOS and Android, that is why for some devices handle button was overlapped